policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/montgomery/match.yaml

```diff
@@ -0,0 +1,11 @@
+description: Montgomery County matches this percentage of the State Earned Income Tax Credit for taxpayers who qualify for the Refundable EITC from the State of Maryland, called the Working Families Income Supplement.
+values:
+
+  2023-07-01: 0.56 # what date??
+metadata:
+  unit: /1
+  label: Montgomery Refundable State EITC match
+  reference: # law reference??
+    - title: Montgomery County Earned Income Credit EIC Refund
+      href: https://www3.montgomerycountymd.gov/311/Solutions.aspx?SolutionId=1-4DAM0I
```

Comment:
2016 and prior law:
https://codelibrary.amlegal.com/codes/montgomerycounty/latest/montgomeryco_md/0-0-0-129735

It seems like it was not updated with the new bill 
I am reading this as a full match but 90% in 2025 and 95% in 2026 


But it definitely is 56% starting in 2023/07 - need to find more up to date bill